docs/model-configuration.md

```diff
@@ -0,0 +1,66 @@
+1. `input_size`: List - image resolution which is going to be processed. Reshapes network to match a given size
+##### `FaceBoxes`
+1. `iou_threshold`: float - threshold for non-maximum suppression (NMS) intersection over union (IOU) filtering
+##### `NanoDet`
+1. `iou_threshold`: float - threshold for non-maximum suppression (NMS) intersection over union (IOU) filtering
+1. `num_classes`: int - number of classes
+##### `UltraLightweightFaceDetection`
+1. `iou_threshold`: float - threshold for non-maximum suppression (NMS) intersection over union (IOU) filtering
+##### `YOLO` and its subclasses
+1. `iou_threshold`: float - threshold for non-maximum suppression (NMS) intersection over union (IOU) filtering
+###### `YoloV4`
+1. `anchors`: List - list of custom anchor values
+1. `masks`: List - list of mask, applied to anchors for each output layer
+###### `YOLOX`
+1. `iou_threshold`: float - threshold for non-maximum suppression (NMS) intersection over union (IOU) filtering
+#### `HpeAssociativeEmbedding`
+1. `target_size`: int - image resolution which is going to be processed. Reshapes network to match a given size
+1. `aspect_ratio`: float - image aspect ratio which is going to be processed. Reshapes network to match a given size
+1. `confidence_threshold`: float - pose confidence threshold
+1. `delta`: float
+1. `size_divisor`: int - width and height of the rehaped model will be a multiple of this value
+1. `padding_mode`: str - center or right_bottom
+#### `OpenPose`
+1. `target_size`: int - image resolution which is going to be processed. Reshapes network to match a given size
+1. `aspect_ratio`: float - image aspect ratio which is going to be processed. Reshapes network to match a given size
+1. `confidence_threshold`: float - pose confidence threshold
+1. `upsample_ratio`: int - upsample ratio of a model backbone
+1. `size_divisor`: int - width and height of the rehaped model will be a multiple of this value
+#### `MaskRCNNModel`
+1. `confidence_threshold`: float - probability threshold value for bounding box filtering
+1. `labels`: List - list of class labels
+1. `path_to_labels`: str - path to file with labels. Overrides the labels, if they sets via `labels` parameter
+#### `SegmentationModel` and its subclasses
+1. `labels`: List - list of class labels
+1. `path_to_labels`: str - path to file with labels. Overrides the labels, if they sets via 'labels' parameter
+### `Bert` and its subclasses
+1. `vocab`: Dict - mapping from string token to int
+1. `input_names`: str - comma-separated names of input layers
+1. `enable_padding`: bool - should be input sequence padded to max sequence len or not
+#### `BertQuestionAnswering`
+1. `output_names`: str - comma-separated names of output layers
+1. `max_answer_token_num`: int
+1. `squad_ver`: str - SQuAD dataset version used for training. Affects how postprocessing
+
+> Geti Anomaly model wrapper adds `image_threshold`, `pixel_threshold`, `min`, `max`, `threshold`.
```

Comment:
It's better to refer to Anomalib or OTX from the note since they are public. BTW, OV documentation uses the following markdown sequence for notes: `> **NOTE**:` (with the space symbol)